tests/e2e/pageobjects/dashboard/Workspaces.ts

```diff
@@ -17,7 +17,7 @@ import { TimeoutConstants } from '../../TimeoutConstants';
 
 export enum WorkspaceStatusUI {
     Running = 'green',
-    Stopped = 'grey'
+    Stopped = 'grey',
```

Comment:
@makambalaji this doesn't look right. There are no more variables after this one, the `,` shouldn't be there.

Reply:
fixed with the linter code change